### ISSUE TITLE
Validation set and Test set

### ISSUE THREAD

**Issue by ANLGBOY** [stevenygd/PointFlow#11]
It seems that the validation set used In train.py and the test set used in test.py are the same since they are obtained through the same function 'get_datasets' in datasets.py

I'm wondering whether the authors used the test set or the validation for the reported result in the paper of PointFlow.

**Comment by stevenygd** [comment]
The results on paper is reported using the code released.
